scripts/cassandra-integration-test.sh

```diff
@@ -9,13 +9,25 @@ export CASSANDRA_USERNAME="cassandra"
 export CASSANDRA_PASSWORD="cassandra"
 success="false"
 
+SKIP_APPLY_SCHEMA="false"
+
+while getopts "s" opt; do
+	case "${opt}" in
+	s)
+    SKIP_APPLY_SCHEMA="true"
+		;;
+  *)
+		;;
```

Comment:
please use two-space indentation, not tabs

Reply:
Fixed it. Thanks